src/components/Header/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import styles from "./styles.module.css";

```

Comment:
Same comment about import orders (apply this idea elsewhere, I won't comment on each file even if they are affected too).